### PR TITLE
docs(BugReport): Add issue template

### PR DESCRIPTION
**Pull request by ElijahKotyluk**
### Bug/Issue template [WIP]

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+### Actual Behavior
+<!-- A clear and concise description of what the bug is. -->
+
+### Steps to reproduce
+<!-- Steps to reproduce the bug you are experiencing. -->
+
+### Expected Behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+### Reproduction Link
```
Comment (dsseng) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: this should allow single-file repros too, as well as online playgrounds

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+name: Bug report
+about: Create a bug report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+### Environment
+**OS:** [e.g. iOS]
```
Comment (dsseng) [inline .github/ISSUE_TEMPLATE/bug_report.md]: not a good example for nodejs

**Review by dsseng — COMMENTED** [review]
What's about feature request template?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ assignees: ''
 
 ### Reproduction Link
 <!--
-Please provide a minimal repository on GitHub that reproduces the issue or bug you are experiencing.
+Please provide a minimal repository, gist, or online playground(Repl.it) that reproduces the issue or bug you are experiencing.
```
Comment (dsseng) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Inline code in issue is acceptable, probably should be at the first place

Comment (ElijahKotyluk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Ah how could I forgot inline code. I’ll also set up a playground to use with packages pre installed once first version is complete and will replace link.

Comment (dsseng) [reply .github/ISSUE_TEMPLATE/bug_report.md]: great idea!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (dsseng) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
**OS:** [e.g. macOS]
```
Any Linux distro may be good as well.

**Review by dsseng — APPROVED** [review]

**Pull request merged**